app/services/jwtService.ts

```diff
@@ -4,6 +4,7 @@ export interface JwtPayload {
   userId: number;
   emailConfirmed: boolean;
   role: string;
+  blocked: boolean;
```

Comment:
Tak przechowywany status bana powoduje, że 'blocked' w tokenie jest ważne aż do wygaśnięcia tokenu. Jeśli token wygasa po 30 dniach i zbanujesz użytkownika po pięciu dniach, to będzie mógł wprowadzać zmiany jeszcze przez 25 dni

Reply:
Ale przechowywać i tak będę, żebyś na froncie mógł jakiś komunikat wyświetlić